bin/templates/scripts/cordova/lib/podMod.js

```diff
@@ -0,0 +1,162 @@
+var fs = require('fs');
+var util = require('util');
+var events = require('cordova-common').events;
+var superspawn = require('cordova-common').superspawn;
+var CordovaError = require('cordova-common').CordovaError;
+var opts = {};
+/*
+-- After pods are installed in a .xcworkspace, all existing ios code needs to go into the WORKSPACE file -- will need to 
+    create a workspace file and then embed the Xcode project  
+
+        - Holly might have done some work on this, see the docs: 
+          https://github.com/phonegap/phonegap-webview-ios not sure how applicable it can be to our case
+*/
+function removeProjectFromPath (pathToProjectFile) {
+    var array = [];
+    //remove the project from the path
+    array = pathToProjectFile.split('/');
+    array.pop();
+    var path = array.join('/');
+    return path;
+}
+
+function createPodfile (projectName, pathToProjectFile) {
+    var path = removeProjectFromPath(pathToProjectFile);
+    var pathToPodfile = path + '/Podfile';
+    var podfileText = util.format('platform :ios, \'8.0\'\n\ntarget \'%s\' do\n\n  project \'%s\'\n\n  \n\nend' , projectName, pathToProjectFile);
+    fs.writeFileSync(pathToPodfile, podfileText);
+}
+
+function editPodfileSync (Podfile, pod, isRemoval) {
+    var podfileContents = fs.readFileSync(Podfile, 'utf8');
+    //split by \n, add in the pod after the project line, shift the rest down
+    var podfileContentsArray = podfileContents.split('\n');
+    var linesInPodfileToKeep = [];
+
+    if (isRemoval) {
+        podfileContentsArray.forEach (function (lineInPodfile) {
+            //if the line in Podfile is the pod to rm, don't save it to new array (to be returned)
+            if (!lineInPodfile.includes(pod)){
+                linesInPodfileToKeep.push(lineInPodfile); 
+            }
+        });
+        podfileContents = linesInPodfileToKeep.join('\n');
+    } else {
+        podfileContentsArray.splice(5, 0, pod);
```

Comment:
5 is the line in the Podfile that I am injecting the new pod. Added a var to improve readability.